client/post-editor/editor-confirmation-sidebar/index.jsx

```diff
@@ -38,6 +39,21 @@ class EditorConfirmationSidebar extends React.Component {
 		this.props.onPublish( true );
 	};
 
+	getBusyButtonLabel( publishButtonStatus ) {
+		switch ( publishButtonStatus ) {
+			case 'update':
+				return this.props.translate( 'Updating...' );
```

Comment:
It does, but @a8ci18n wants three dots because there's already an existing string "Updating...", I figured not created yet another string that wouldn't be translated was the better optimization.